src/Sundials.jl

```diff
@@ -1,4 +1,7 @@
 module Sundials
+
+using Compat
```

Comment:
you should list a minimum version in REQUIRE that implements the features you're using